### PR TITLE
feat(login): add new user login feature

### LINKED ISSUE #3940: Feat: login social logos button improved version
### Is your feature request related to a problem? Please describe

the social icons shown in the login can be improved .Instead of just showing the icons we can also improve their view '

### Describe the solution you'd like

Initially how the social logos looks 
Screenshot (184)

How they can be improved
Screenshot (187)

### Describe alternatives you've considered

_No response_

### Additional context

so this helps users to get proper view and also provide familarity with login options 

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by harshsharma-11**
fixes #3940

#### Describe the changes you have made in this PR -
I have made the social logos to look more attractive and aligned in better way and also add text about the logos 
### Screenshots of the changes (If any) -

Screenshot (191)

Note :please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -65,6 +65,52 @@ body {
   text-align: center;
 }
 
+.log-cont{
+  text-decoration: none;
```
[the remark this answers is not in the document]
Comment (harshsharma-11) [reply app/assets/stylesheets/layout.scss:69]: don't get it ,need more clarification

**Comment by harshsharma-11** [comment]
fixes#3940

**Comment by harshsharma-11** [comment]
please review this,if any problem in this i am still here to improve it please check this .......................................

**Comment by harshsharma-11** [comment]
Hey @abstrekt, could you please take a look at this issue and review my pr

**Comment by harshsharma-11** [comment]
@tanmoysrt please review my pr,waiting from 4 days

**Comment by tanmoysrt** [comment]
> @tanmoysrt please review my pr,waiting from 4 days

cc @vedant-jain03

**Comment by harshsharma-11** [comment]
cc @tanmoysrt i am waiting for your review

**Comment by vedant-jain03** [comment]
Hi @harshsharma-11 thanks for the interest and raising PR. There are certain steps and things you should do before raising a PR:
- Validate if the feature or fix raised by use is assigned to you
- If you have reqeusted a new issue, you should have first take approval from core-team. So that we can direct you in a better way.
- For Ui/UX changes, you first need to have approval from team in slack under #ux channel. This point also mentioned in your thread https://github.com/CircuitVerse/CircuitVerse/issues/3940#issuecomment-1676419524


Review on this PR:
- I don't think so we need to change the existing UI, it looks perfect to me.

Closing this PR, try to adapt points I mentioned above, thanks for your contribution. You can always reach out for help in slack channel. Thankyou 🚀

**Pull request closed without merging**